packages/forms/src/components/FilterMenu/index.tsx

```diff
@@ -54,6 +56,7 @@ export default function FilterMenu({
   keepOpenOnClear,
   large,
   menuProps,
+  mutedReset = true,
```

Comment:
usually boolean value's default is `false`. please rename this to `unmuted`, default of `false` then `muted={!unmuted}`